STANDARD OUT:


STANDARD ERROR:
/home/yxw190015/tvb-pipeline2/tvb-ukbb/bb_functional_pipeline/bb_FC: line 25: mri_segstats: command not found
Traceback (most recent call last):
  File "/home/yxw190015/tvb-pipeline2/tvb-ukbb/bb_functional_pipeline/bb_FC_compute", line 75, in <module>
    main()
  File "/home/yxw190015/tvb-pipeline2/tvb-ukbb/bb_functional_pipeline/bb_FC_compute", line 47, in main
    TS_old = np.loadtxt(ts_rois)
  File "/home/yxw190015/tvb-pipeline2/software/condaenv/lib/python3.7/site-packages/numpy/lib/npyio.py", line 1065, in loadtxt
    fh = np.lib._datasource.open(fname, 'rt', encoding=encoding)
  File "/home/yxw190015/tvb-pipeline2/software/condaenv/lib/python3.7/site-packages/numpy/lib/_datasource.py", line 194, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/yxw190015/tvb-pipeline2/software/condaenv/lib/python3.7/site-packages/numpy/lib/_datasource.py", line 531, in open
    raise IOError("%s not found." % path)
OSError: /home/yxw190015/HSAM/000017/fMRI/rfMRI.ica/ts_roied.txt not found.
